maths/SieveOfEratosthenesPrimeNumber.py

```diff
@@ -0,0 +1,29 @@
+'''
+Sieve of Eratosthenes
+
+Input : n =10
+Output : 2 3 5 7 
+
+Input : n = 20 
+Output: 2 3 5 7 11 13 17 19
+'''
+
+def primeNumbers(num):
+    
```

Comment:
Add doctests as discussed in CONTRIBUTING.md:
```python
def prime_numbers(num: int) -> None:
    """
    >>> prime_numbers(10)
    2 3 5 7 
    >>> prime_numbers(20)
    2 3 5 7 11 13 17 19 
    """
```

Reply:
added that